tutorials/outflux.md

```diff
@@ -0,0 +1,174 @@
+* Extract it to a preferred location 
+
+If you navigate to where you extracted the archive and execute:
+
+```bash
+$ ./outflux --help
+Outflux offers the capabilities to migrate an InfluxDB database, or specific measurements to TimescaleDB.
+It can also allow a user to transfer only the schema of a database or measurement to TimescaleDB
+
+Usage:
+  outflux [command]
+
+Available Commands:
+  help            Help about any command
+  migrate         Migrate the schema and data from InfluxDB measurements into TimescaleDB hypertables
+  schema-transfer Discover the schema of measurements and validate or prepare a TimescaleDB hyper-table
+                  with the discovered schema
+```
+
+You will see the help output for Outflux, a brief explanation of what it can do, the usage, and available commands. 
+
+For instructions on how to set up Outflux from source you can visit the [README](https://github.com/timescale/outflux/blob/master/README.md).
+
+## Importing Sample Data
+
+If you don’t already have an existing InfluxDB database, you can try Outflux by importing this example file with data written in the Influx Line Protocol found at https://timescaledata.blob.core.windows.net/datasets/outflux_taxi.txt 
+
+You can use the Influx CLI client to load the data. The file will first create the “outflux_tutorial” database and then do 2741 inserts. 
+
+```bash
+$ influx -import -path=outflux_taxit.txt -database=outflux_tutorial
+2019/03/27 11:39:11 Processed 1 commands
+2019/03/27 11:39:11 Processed 2741 inserts
+2019/03/27 11:39:11 Failed 0 inserts
+```
+
+The data in the file is without a timestamp so the current time of the Influx server will be used at the time of insert. All the data points belong to one measurement `taxi`. The points are tagged with location, rating and vendor. Four fields are recorded: fare, mta_tax, tip and tolls. 
+The `influx` client assumes the server is available at http://localhost:8086 by default.
+
+## Schema Discovery, Validation and Transfer
+
+One of Outflux’s features is the ability to discover the schema of an InfluxDB measurement and either validate that a TimescaleDB table exists that can hold the transferred data, or create a new table that can satisfy the schema requirements. 
+
+We can now create a TimescaleDB hypertable ready to receive the demo data we inserted into the InfluxDB instance. If you followed the tutorial and inserted the data from the example, there should be a `taxi` measurement in the `outflux_tutorial` database in the InfluxDB instance. 
+
+The `schema-transfer` command of Outflux can work with 4 schema strategies: 
+* `ValidateOnly`: checks if a specified TimescaleDB database has a hypertable with the proper columns and if it’s partitioned properly, but will not perform modifications
+* `CreateIfMissing`: validates everything if it exists, and it will check if the TimescaleDB  extension is installed (if the hypertable is missing, it will create it  and partition it properly)
```

Comment:
Not clear here...does `ValidateOnly` also check if the TimescaleDB extension is installed?  If so, maybe that should be mentioned under `ValidateOnly` bullet and `CreateIfMissing` should say for example: "Runs all checks that `ValidateOnly` does and creates and properly partitions any missing hypertables."